cmake/wsrep.cmake

```diff
@@ -29,6 +29,8 @@ ENDIF()
 # is moved to its own wsrep_version.cmake file.
 
 SET(WSREP_C_FLAGS   "-DWITH_WSREP -DWSREP_PROC_INFO -DMYSQL_MAX_VARIABLE_VALUE_LEN=4096")
+ADD_DEFINITIONS(-DWSREP_PS_VAR_VALUE_BUFF_SIZE=4096)
```

Comment:
Could you please add a comment here which sql files (and where) we need to change if we change the value here? (we've got back-link from sql to here, but the other way may be useful as well I think)

Reply:
Done (I'll rebase 8.0 PR after approving this one)